scripts/ci/osx/build.sh

```diff
@@ -3,3 +3,19 @@ set -ev
 echo "Building openFrameworks - OSX Template Project"
 ROOT=${TRAVIS_BUILD_DIR:-"$( cd "$(dirname "$0")/../../.." ; pwd -P )"}
 xcodebuild -configuration Release -target emptyExample -project "$ROOT/scripts/osx/template/emptyExample.xcodeproj"
+
+echo "Unit tests"
+cd $ROOT/tests
+for group in *; do
+	if [ -d $group ]; then
+		for test in $group/*; do
+			if [ -d $test ]; then
+				cd $test
+				cp ../../../scripts/osx/template/Makefile .
+				cp ../../../scripts/osx/template/config.make . 
+				make Debug
```

Comment:
i know, i'll change it when i have time but if i use xcodebuild i need to compile the PG, generate the project for the tests and then compile them so it's not trivial. i'll probably move the PG generation here if i finally do it like that